addOns/postman/src/main/java/org/zaproxy/addon/postman/models/Body.java

```diff
@@ -0,0 +1,215 @@
+        this.raw = raw;
+    }
+
+    public List<KeyValueData> getUrlencoded() {
+        return urlencoded;
+    }
+
+    public void setUrlencoded(List<KeyValueData> urlencoded) {
+        this.urlencoded = urlencoded;
+    }
+
+    public List<FormData> getFormData() {
+        return formData;
+    }
+
+    public void setFormData(List<FormData> formData) {
+        this.formData = formData;
+    }
+
+    public File getFile() {
+        return file;
+    }
+
+    public void setFile(File file) {
+        this.file = file;
+    }
+
+    public Object getGraphql() {
+        return graphql;
+    }
+
+    public void setGraphql(GraphQL graphql) {
+        this.graphql = graphql;
+    }
+
+    public Options getOptions() {
+        return options;
+    }
+
+    public void setOptions(Options options) {
+        this.options = options;
+    }
+
+    public boolean isDisabled() {
+        return disabled;
+    }
+
+    public void setDisabled(boolean disabled) {
+        this.disabled = disabled;
+    }
+
+    public static class File {
+        @JsonDeserialize(using = ObjectDeserializer.class)
+        private String src;
+
+        public String getSrc() {
+            return src;
+        }
+
+        public void setSrc(String src) {
+            this.src = src;
+        }
+    }
+
+    public static class FormData extends KeyValueData {
+        @JsonDeserialize(using = ObjectDeserializer.class)
+        private String src;
+
+        @JsonDeserialize(using = ObjectDeserializer.class)
+        private String type;
+
+        public String getSrc() {
+            return src;
+        }
+
+        public void setSrc(String src) {
+            this.src = src;
+        }
+
+        public String getType() {
+            return type;
+        }
+
+        public void setType(String type) {
+            this.type = type;
+        }
+    }
+
+    public static class GraphQL {
+        @JsonDeserialize(using = ObjectDeserializer.class)
+        private String query;
+
+        @JsonDeserialize(using = ObjectDeserializer.class)
+        private String variables;
+
+        public String getQuery() {
+            return query;
+        }
+
+        public void setQuery(String query) {
+            this.query = query;
+        }
+
+        public String getVariables() {
+            return variables;
+        }
+
+        public void setVariables(String variables) {
+            this.variables = variables;
+        }
+    }
+
+    public static class Options {
+        @JsonDeserialize(using = ObjectDeserializer.class)
+        private RawOption raw;
+
+        public RawOption getRaw() {
+            return raw;
+        }
+
+        public void setRaw(RawOption raw) {
+            this.raw = raw;
+        }
+
+        public static class RawOption {
+            @JsonDeserialize(using = ObjectDeserializer.class)
+            private String language;
+
+            public RawOption(String language) {
+                this.language = "text";
```

Comment:
`… = language;`? Would be good to have tests deserializing all these objects (though I expect that to be added later so not a requirement).